src/Notes/WooCommercePayments.php

```diff
@@ -128,22 +135,58 @@ protected static function is_installed() {
 	}
 
 	/**
-	 * Install WooCommerce Payments when note is actioned.
+	 * Install and activate WooCommerce Payments.
 	 *
-	 * @param Note $note Note being acted upon.
+	 * @return boolean Whether the plugin was successfully activated.
 	 */
-	public function install( $note ) {
-		if ( self::NOTE_NAME === $note->get_name() && current_user_can( 'install_plugins' ) ) {
```

Comment:
I think we should keep at least this check: `current_user_can( 'install_plugins' )`

Reply:
Haven't noticed the check below, sorry.